bigtable-hbase-1.x-parent/bigtable-hbase-1.x-integration-tests/pom.xml

```diff
@@ -323,68 +363,106 @@ limitations under the License.
           <artifactId>org.apache.hbase</artifactId>
           <groupId>*</groupId>
         </exclusion>
-        <exclusion>
-          <artifactId>${project.groupId}</artifactId>
-          <groupId>*</groupId>
-        </exclusion>
       </exclusions>
     </dependency>
 
+
+    <!-- java-bigtable Modules -->
+    <dependency>
+      <groupId>com.google.cloud</groupId>
+      <artifactId>google-cloud-bigtable</artifactId>
+      <scope>test</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>com.google.api.grpc</groupId>
+      <artifactId>proto-google-cloud-bigtable-v2</artifactId>
+      <scope>test</scope>
+    </dependency>
+
+
+    <!-- HBase testing tools -->
+    <dependency>
+      <groupId>org.apache.hbase</groupId>
+      <artifactId>hbase-shaded-testing-util</artifactId>
+      <version>${hbase.version}</version>
+      <scope>test</scope>
+    </dependency>
+
+
+    <!-- Misc -->
+    <dependency>
+      <groupId>com.google.auth</groupId>
+      <artifactId>google-auth-library-credentials</artifactId>
+      <scope>test</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>com.google.auth</groupId>
+      <artifactId>google-auth-library-oauth2-http</artifactId>
+      <scope>test</scope>
+    </dependency>
+
     <dependency>
       <groupId>com.google.code.findbugs</groupId>
       <artifactId>jsr305</artifactId>
       <version>${jsr305.version}</version>
       <scope>test</scope>
     </dependency>
+
     <dependency>
       <groupId>com.google.guava</groupId>
       <artifactId>guava</artifactId>
-      <version>${hbase-guava.version}</version>
```

Comment:
was this the wrong version?

Reply:
No, but the context has changed.
Previously integration tests would use a shaded version of bigtable client and an expanded version of hbase minicluster, so we had to use hbase's version of guava.
Now bigtable is expanded and hbase is shaded, so we get to use bigtable's version of guava